target/linux/ath79/generic/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -78,6 +78,9 @@ case "$board" in
 		[ "$PHYNBR" -eq 1 ] && \
 			macaddr_add "$(mtd_get_mac_ascii u-boot-env ethaddr)" 1 > /sys${DEVPATH}/macaddress
 		;;
+	kuwfi,n650)
```

Comment:
You can do this in dts with Mac-base

Reply:
Hm, didn't work for me. Could you share an example?

Reply:
https://github.com/openwrt/openwrt/commit/b7f26c63929da908f83c767689bc25ef161db466

Reply:
Thanks, I'll fix that. I was probably too tired when I crafted the DTS.